Helper/OrderSerializer.php

```diff
@@ -31,15 +32,26 @@ public function serialize($order) {
             "address"       => is_array($street) ? implode(PHP_EOL, $street) : $street,
             "city"          => $orderBillingData->getCity(),
             "countryCode"   => $orderBillingData->getCountryId(),
-            "phone"         => $orderBillingData->getTelephone(),
+            "phone"         => $orderPhone,
             "postcode"      => $orderBillingData->getPostcode(),
             "paymentMethod" => $order->getPayment()->getMethodInstance()->getTitle()
         ];
-    
+        
+        if (empty($order->getCustomerEmail())) {
+            if (isset($orderPhone)) {
```

Comment:
`$orderPhone` will always be set, but it could be empty.

Reply:
Hmm, that is correct. Updated the logic.